modules/340-monitoring-deckhouse/charts/helm_lib/templates/_module_name.tpl

```diff
@@ -0,0 +1,3 @@
+{{- define "helm_lib_module_camelcase_name" -}}
```

Comment:
Helm lib is a copy, not a symlink? AFIK, it should be a symlink.

Reply:
Ups, IDE magic %)

Fixed